build.assets/Dockerfile-centos7-assets

```diff
@@ -1,17 +1,17 @@
 # syntax=docker/dockerfile:1
 
-FROM centos:7 AS centos-devtoolset
+FROM --platform=$BUILDPLATFORM centos:7 AS centos-devtoolset
```

Comment:
Same `--platform` question as above: are we actually using it?

Reply:
This starts the process of allowing us to do multi-architecture builds. It is not used yet, as we need to update all compilation to specify the correct target. See https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/ for a good overview on how this works.